doc/api/api_changes/2017-12-06-KL.rst

```diff
@@ -0,0 +1,13 @@
+Change return value of Axes.get_shared_[x,y,z]_axes()
+-----------------------------------------------
+
+The return of get_shared_x_axes() and respectively for y and z axis
+returned a Grouper object. Now it returns a WeakSet object.
+
+Workarounds:
+* If the intention is to get siblings as previous then the WeakSet contains all the siblings,
+then `list(r)` where r is the returned value is sufficient.
+
```

Comment:
These three points would be easier to understand with small code snippets:  i.e. 

```python
sharedx = ax.get_shared_x_axes()
print(sharedx.sibblings)
```
can now be acieved with:
```python
sharedx = ax.get_shared_x_axes()
print(list(sharedx))
```